collections/nemo_asr/nemo_asr/parts/dataset.py

```diff
@@ -1,17 +1,16 @@
 # Audio dataset and corresponding functions taken from Patter
 # https://github.com/ryanleary/patter
 # TODO: review, and copyright and fix/add comments
-import kaldi_io
 import os
-import pandas as pd
-import string
+
+import kaldi_io
 import torch
+from nemo_asr.parts import char_parsers
+from nemo_asr.parts import manifests
 from torch.utils.data import Dataset
```

Comment:
Can you move this import line above the rest?

Reply:
That's how pycharm reoder imports.

Reply:
Please do Standard library imports first then other import than all nemo imports.

Reply:
OK.
Dont you think it's kinda pointless until we somehow enforce it? There are no standardise order at this point.
I can research tools options for enforcing order and include it with black PR.

Reply:
Yes it would be nice to enforce it somehow. Right now, it's all manual.

Reply:
Done.